fsr/FastADC.h

```diff
@@ -0,0 +1,19 @@
+// defines for setting and clearing register bits
```

Comment:
I think the stuff in this file is small enough that it can just be in the main ino for now (and move the file location back to the original location. I do plan on splitting up the file a bit later down the road.

Reply:
Ahh ok, i figured that a lot of the features needed to be double wrapped (e.g. EEpromEX for ATmega vs teensy is different) and a bunch of those C++ classes could move. But if you want to do that on your own i'll move everything back.